src/Domain/Creators/AlignedLattice.hpp

```diff
@@ -31,6 +46,18 @@ namespace creators {
 template <size_t VolumeDim>
 class AlignedLattice : public DomainCreator<VolumeDim> {
  public:
+  using maps_list = tmpl::list<
```

Comment:
shouldn't this list depend upon VolumeDim?

Reply:
It could, but as far as I can tell registering something twice doesn't seem to matter. I can double check with the Charm folks. My reason for doing it this way was to avoid a double nested `tmpl::conditional_t` because I found that a bit ugly.

Reply:
One fairly easy way to get around double registration is changing the way the registration function works. I'll do that and then we don't have to worry about double registering anything.